keyboards/nuphy/air75_v2/ansi/ansi.c

```diff
@@ -0,0 +1,729 @@
+                f_rf_sw_press = 0;
+
+                if (rf_sw_press_delay < RF_LONG_PRESS_DELAY) {
+                    dev_info.link_mode   = rf_sw_temp;
+                    dev_info.rf_channel  = rf_sw_temp;
+                    dev_info.ble_channel = rf_sw_temp;
+                    uart_send_cmd(CMD_SET_LINK, 10, 20);
+                }
+            }
+            return false;
+
+        case LNK_BLE1:
+            if (record->event.pressed) {
+                if (dev_info.link_mode != LINK_USB) {
+                    rf_sw_temp    = LINK_BT_1;
+                    f_rf_sw_press = 1;
+                    break_all_key();
+                }
+            } else if (f_rf_sw_press) {
+                f_rf_sw_press = 0;
+
+                if (rf_sw_press_delay < RF_LONG_PRESS_DELAY) {
+                    dev_info.link_mode   = rf_sw_temp;
+                    dev_info.rf_channel  = rf_sw_temp;
+                    dev_info.ble_channel = rf_sw_temp;
+                    uart_send_cmd(CMD_SET_LINK, 10, 20);
+                }
+            }
+            return false;
+
+        case LNK_BLE2:
+            if (record->event.pressed) {
+                if (dev_info.link_mode != LINK_USB) {
+                    rf_sw_temp    = LINK_BT_2;
+                    f_rf_sw_press = 1;
+                    break_all_key();
+                }
+            } else if (f_rf_sw_press) {
+                f_rf_sw_press = 0;
+
+                if (rf_sw_press_delay < RF_LONG_PRESS_DELAY) {
+                    dev_info.link_mode   = rf_sw_temp;
+                    dev_info.rf_channel  = rf_sw_temp;
+                    dev_info.ble_channel = rf_sw_temp;
+                    uart_send_cmd(CMD_SET_LINK, 10, 20);
+                }
+            }
+            return false;
+
+        case LNK_BLE3:
+            if (record->event.pressed) {
+                if (dev_info.link_mode != LINK_USB) {
+                    rf_sw_temp    = LINK_BT_3;
+                    f_rf_sw_press = 1;
+                    break_all_key();
+                }
+            } else if (f_rf_sw_press) {
+                f_rf_sw_press = 0;
+
+                if (rf_sw_press_delay < RF_LONG_PRESS_DELAY) {
+                    dev_info.link_mode   = rf_sw_temp;
+                    dev_info.rf_channel  = rf_sw_temp;
+                    dev_info.ble_channel = rf_sw_temp;
+                    uart_send_cmd(CMD_SET_LINK, 10, 20);
+                }
+            }
+            return false;
+
+        case MAC_TASK:
+            if (record->event.pressed) {
+                host_consumer_send(0x029F);
+            } else {
+                host_consumer_send(0);
+            }
+            return false;
+
+        case MAC_SEARCH:
+            if (record->event.pressed) {
+                register_code(KC_LGUI);
+                register_code(KC_SPACE);
+                wait_ms(50);
+                unregister_code(KC_LGUI);
+                unregister_code(KC_SPACE);
+            }
```

Comment:
this is just `LGUI(KC_SPC)` (or `G(KC_SPC)`.  No need to add a separate keycode for this.